aya/src/maps/mod.rs

```diff
@@ -42,7 +42,7 @@ use std::{
     marker::PhantomData,
     mem,
     ops::Deref,
-    os::fd::{AsFd, AsRawFd, OwnedFd, RawFd},
+    os::fd::{AsFd as _, AsRawFd, IntoRawFd as _, OwnedFd, RawFd},
```

Comment:
```suggestion
    os::fd::{AsFd as _, AsRawFd as _, IntoRawFd as _, OwnedFd, RawFd},
```

Reply:
This trait is explicitly used in this module for `impl AsRawFd for MapFd`.